lib/carraway/file.rb

```diff
@@ -3,13 +3,14 @@
 
 module Carraway
   class File
-    attr_reader :uid, :created, :file, :published
+    attr_reader :uid, :ext, :created, :file, :published
     attr_accessor :title, :labels, :category
 
-    def initialize(title:, file: nil, uid: nil, created: Time.now.to_i, labels: nil, published: nil, category: nil)
+    def initialize(title:, file: nil, uid: nil, ext: nil, created: Time.now.to_i, labels: nil, published: nil, category: nil)
       @title = title
       @file = file
       @uid = uid || generate_uid
+      @ext = ext || file && file[:filename]&.split('.')&.last
```

Comment:
Previous records do not have ext attribute, so that makes `nil` ext records.
I think `.pdf` is necessary as a default value.